src/sidecar/middlewares.ts

```diff
@@ -1,7 +1,6 @@
 import * as vscode from "vscode";
 import { Middleware, RequestContext, ResponseContext } from "../clients/sidecar";
 
-import { ccloudAuthSessionInvalidated } from "../emitters";
```

Comment:
Unused import, unrelated to the body of this PR but found while testing out the `pre` middleware idea in the PR description